resources/guides/Navigation Guide/sitemap.json

```diff
@@ -0,0 +1,268 @@
+{
+  "Home": [
+    {
+      "name": "Home",
+      "loc": "https://uwindsorcss.github.io/wiki"
```

Comment:
The `loc` having the full wiki url is an issue. It is an issue because when we change the domain, it will still point to that one. at the end of the day this is an issue but one with no easy solution.